app/views/users/index.html.erb

```diff
@@ -13,11 +13,9 @@
 
 <% @users.each do |user| %>
   <tr>
-    <td><%= user.name %></td>
+    <td><%= link_to user.name, user_path(user) %></td>
```

Comment:
Since there's nothing to show for Terms, the only relevant action is Edit, and that action currently is overlaid on the term name itself.  For users, all three actions are relevant, and so moving Show onto the user name itself would make it inconsistent with the terms page.